lib/iris/tests/results/analysis/addition_coord_x.cml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" ?>
 <cubes xmlns="urn:x-iris:cubeml-0.2">
-  <cube core-dtype="float64" dtype="float64" fill_value="-1e+30" units="K">
+  <cube core-dtype="float64" dtype="float64" fill_value="-1.00000001505e+30" units="K">
```

Comment:
> Are these CML files compared textually in the code? If so, it's rather worrying that we have floating points values in there...

Yes, but that's inevitable in an XML representation.
It's really only a tiny part of what is wrong with assertCML ...